packages/beacon-node/src/network/peers/peerManager.ts

```diff
@@ -620,7 +620,10 @@ export class PeerManager {
       },
       {retries: 3, retryDelay: 1000}
     ).catch((err) => {
-      this.logger.error("Error setting agentVersion for the peer", {peerId: peerData.peerId.toString()}, err);
+      // Ignore Not Found error if peer was disconnected
+      if (err instanceof Error && !err.message.includes("NotFound")) {
+        this.logger.error("Error setting agentVersion for the peer", {peerId: peerData.peerId.toString()}, err);
```

Comment:
I think we only want to log with debug level here, and no need to check for "NotFound"

I think the cause is https://github.com/ChainSafe/lodestar/issues/6382

should work on that instead